src/txmempool.cpp

```diff
@@ -1055,11 +1055,10 @@ void CTxMemPool::TrimToSize(size_t sizelimit, std::vector<COutPoint>* pvNoSpends
     }
 }
 
-bool CTxMemPool::TransactionWithinChainLimit(const uint256& txid, size_t chainLimit) const {
+int64_t CTxMemPool::chain_limit_value(const uint256& txid) const {
```

Comment:
Snake case is for variables.

Reply:
It seems like we are moving to snake case, e.g. for new code like

https://github.com/bitcoin/bitcoin/blob/a34ac6ae0788b50e12dd2c8ac59dbda2a03a3c2e/src/cuckoocache.h#L92

Maybe I'm mistaken on that one. Will check recent merges.

Reply:
From the developer notes:
  - Variable and namespace names are all lowercase, and may use `_` to
    separate words (snake_case).
  - Class names, function names and method names are UpperCamelCase
    (PascalCase).

Reply:
Wow, okay. I totally missed that, thanks!

Reply:
Fixed.